website/api/controllers/deliver-apple-csr.js

```diff
@@ -139,7 +139,7 @@ module.exports = {
       template: 'email-signed-csr-for-apns',
       templateData: {},
       attachments: [{
-        contentBytes: generateCertificateResult.request,
+        contentBytes: Buffer.from(generateCertificateResult.request).toString('base64'),
```

Comment:
Let's add a comment for why this is done please.

Reply:
good idea, thanks! Please let me know if the wording makes sense.